src/org/opendatakit/briefcase/ui/FormTransferTable.java

```diff
@@ -26,13 +26,8 @@
 import java.util.List;
 import java.util.Map;
 
-import javax.swing.JButton;
-import javax.swing.JOptionPane;
-import javax.swing.JTable;
-import javax.swing.UIManager;
-import javax.swing.table.AbstractTableModel;
-import javax.swing.table.TableCellRenderer;
-import javax.swing.table.TableColumnModel;
+import javax.swing.*;

```

Comment:
There isn't yet a style guide for this repo but I think that since it has been a convention throughout the code and is in most style guides (for example, [Google's](https://google.github.io/styleguide/javaguide.html#s3.3.1-wildcard-imports)), we will not use wildcard imports. Could you please change these back? I know it's hard to adhere to an invisible style guide, sorry!